package.json

```diff
@@ -34,7 +34,8 @@
     "@babel/plugin-transform-runtime": "^7.1.0",
     "@babel/preset-env": "^7.1.0",
     "@babel/preset-react": "^7.0.0",
-    "babel-plugin-transform-dynamic-import": "^2.1.0"
+    "babel-plugin-transform-dynamic-import": "^2.1.0",
+    "core-js": "3"
```

Comment:
I think we might just need the version entry in the config to disable the warnings, might be worth trying that before we add this.

Reply:
I think so too and upgrading core-js could be a separate task, no?

Reply:
yeah, lets just try it out without this dependency and if we need to add it maybe we make it a peer dep

Reply:
I pushed this update and simply declaring the version as `2`fixed the warning locally 🌟